external-crates/move/crates/move-analyzer/src/symbols/mod.rs

```diff
@@ -368,12 +369,12 @@ pub fn compute_symbols_typed_program(
     CompiledProgram,
 ) {
     // run typing analysis for the main user program
-    let compiler_analysis_info = compiled_pkg_info.compiler_analysis_info.as_ref().unwrap();
```

Comment:
This is the old `unwrap` that I mentioned in https://github.com/MystenLabs/sui/pull/24704/files#r2635378417 that I decided to remove in this PR outside of the PR's core logic as in this position it's tough to argue that it is safe without looking at a completely different part of the code baase